### DISCUSSION TITLE
Single mode for particular dropdown

### DISCUSSION THREAD

**Discussion by abhijeet3112** [xrutayisire/react-js-cron#34] [Q&A]
Is there any functionality we have that supports setting the single dropdown selection for minutes dropdown while keeping other dropdowns with multi select option?

**Comment by xrutayisire** [comment]
Hello,

It's not currently possible, can you explain me a bit more about your use case? 🙂

**Reply by abhijeet3112** [reply]
Hi, 
Actually, my requirement is to trigger a run depending on user inputs but on the same time and at the same time I don't want user to feed an input that runs at different minutes within an hour. Keeping the requirement straight forward, I want minute dropdown to be working in single mode while keeping others in multi mode only.

**Reply by abhijeet3112** [reply]
Is there any planning to have this feature?

**Reply by xrutayisire** [reply]
Hi, it was not planned, but I will try to handle that on the weekend ;)

I think I will update prop "mode" to accept "single", "multiple" or an object with each mode defined for each select.

For example:

```
  mode={{
    months: "multiple",
    month-days: "multiple",
    week-days: "multiple",
    hours: "multiple",
    minutes: "select"
  }}
```

Does this will help you ?

**Reply by xrutayisire** [reply]
If yes, please create an issue for this feature request, so I can have an issue number to track this down ;)

**Reply by abhijeet3112** [reply]
Yes this will help. And I have one more thing. So my requirement is to convert the cron expression as per the timezone. 

For Example: If I wanted to execute a job at 10:00AM IST and for that I have used the proper dropdowns and generated the cron expression as "0 10 * * * ". Now if some one from other timezone go to the UI for that particular jo, he/she will be seeing the same cron expression but I want to get it updated as per the user time zone. Let's say other user is in UTC time zone so he should be shown something like "30 5 * * *".

**Reply by abhijeet3112** [reply]
Created an issue for above feature request.
